product_docs/docs/eprs/6.2/03_installation/03_installing_rpm_package.mdx

```diff
@@ -261,11 +266,13 @@ Successful installation of xDB Replication Server results in the creation of dir
 !!! Note
     Neither the publication server nor the subscription server are running immediately following installation. If after reviewing the remaining steps, you wish to start the publication server, see [Registering a Publication Server](../05_smr_operation/02_creating_publication/01_registering_publication_server/#registering_publication_server). For starting the subscription server see [Registering a Subscription Server](../05_smr_operation/03_creating_subscription/01_registering_subscription_server/#registering_subscription_server).
 
-**Step 5 (For xDB Replication Server 6.2 or 6.1):** In the xDB Replication Configuration file `/etc/edb-repl.conf`, you can either use the default password (`edb`) as the admin user password, or you can substitute a password of your choice. If you want to use your own password, see [Encrypting the Password in the xDB Replication Configuration File](../10_appendix/04_miscellaneous_xdb_processing_topics/02_encrypt_password_inconf_file/) on how to generate the encrypted form of the password. Place the encrypted password in the admin_password parameter of the xDB Replication Configuration file. The default admin user name is set to admin and can be changed as well. See [xDB Replication Configuration File](../02_overview/03_replication_server_components_and_architecture/01_physical_components/#xdb_replication_conf_file) for information on the xDB Replication Configuration file.
+**Step 5:** Install a JDBC driver. See [Installing a JDBC driver](installing_jdbc_driver) for more information.
+
+**Step 6 (For xDB Replication Server 6.2 or 6.1):** In the xDB Replication Configuration file `/etc/edb-repl.conf`, you can either use the default password (`edb`) as the admin user password, or you can substitute a password of your choice. If you want to use your own password, see [Encrypting the Password in the xDB Replication Configuration File](../10_appendix/04_miscellaneous_xdb_processing_topics/02_encrypt_password_inconf_file/) on how to generate the encrypted form of the password. Place the encrypted password in the admin_password parameter of the xDB Replication Configuration file. The default admin user name is set to admin and can be changed as well. See [xDB Replication Configuration File](../02_overview/03_replication_server_components_and_architecture/01_physical_components/#xdb_replication_conf_file) for information on the xDB Replication Configuration file.
```

Comment:
Since Replication Server 5.1 and 6.1 are no longer supported, I wonder if any instructions related to them are needed.  Of the three versions referenced in this and the next step only Replication Server 6.2 is supported and needed to be documented.